packages/sdk-client-v3/tests/client.test/client.test.ts

```diff
@@ -212,69 +212,6 @@ describe('execute function', () => {
         return Promise.resolve()
       })
   })
-
```

Comment:
Why are we removing this tests? 
I think we should have rewritten this test to match the new client implementation.